nixos/modules/services/misc/jellyseerr.nix

```diff
@@ -31,10 +31,10 @@ in
       serviceConfig = {
         Type = "exec";
         StateDirectory = "jellyseerr";
-        WorkingDirectory = "${cfg.package}/libexec/jellyseerr/deps/jellyseerr";
+        WorkingDirectory = "${cfg.package}/share/dist";
         DynamicUser = true;
         ExecStart = lib.getExe cfg.package;
-        BindPaths = [ "/var/lib/jellyseerr/:${cfg.package}/libexec/jellyseerr/deps/jellyseerr/config/" ];
```

Comment:
We could use the `CONFIG_DIRECTORY` environmental variable to accomplish the same, so we don't have to make a bind mount. I'm doing that [here](https://github.com/nix-community/NixNG/pull/53/files#diff-1cf904d1b5b8445ae17d69d2ec5a6fd402a175aef8fe4aff264fbb5e603282fbR45) as well.